README.md

```diff
@@ -46,15 +65,8 @@ If you’re using this demo, please **★Star** this repository to show your int
    ```
 
    Substitute `<PROJECT_ID>` with the ID of your Google Cloud project.
-  
-4. Confirm the services have been enabled for your project.
```

Comment:
Noting that I'm removing this redundant step, since the step above is not async and would tell you immediately if there was an error.

Reply:
I like — shortening time to completion.